src/resource_clients/request_queue.ts

```diff
@@ -425,6 +427,57 @@ export class RequestQueueClient extends ResourceClient {
             }),
         });
     }
+
+    /**
+     * THIS METHOD IS EXPERIMENTAL AND NOT INTENDED FOR PRODUCTION USE.
+     *
+     * @private
+     * @experimental
+     */
+    async listRequests(options: RequestQueueClientListRequestsOptions = {}): Promise<RequestQueueClientListRequestsResult> {
+        ow(options, ow.object.exactShape({
+            limit: ow.optional.number,
+            exclusiveStartId: ow.optional.string,
+        }));
+
+        const response = await this.httpClient.call({
+            url: this._url('requests'),
+            method: 'GET',
+            timeout: this.timeoutMillis,
+            params: this._params({
+                limit: options.limit,
+                exclusiveStartId: options.exclusiveStartId,
+                clientKey: this.clientKey,
+            }),
+        });
+
+        return cast(parseDateFields(pluckData(response.data)));
+    }
+
+    /**
+     * THIS METHOD IS EXPERIMENTAL AND NOT INTENDED FOR PRODUCTION USE.
+     *
+     * Usage:
+     * for await (const { items } of client.paginateRequests({ limit: 10 })) {
+     *   items.forEach((request) => console.log(request));
+     * }
+     * @private
+     * @experimental
+     */
+    paginateRequests(options: RequestQueueClientRequestsOptions = {}): RequestQueueRequestsAsyncIterable<RequestQueueClientListRequestsResult> {
```

Comment:
I think it should be like this:
```suggestion
    paginateRequests(options: RequestQueueClientPaginateRequestsOptions = {}): RequestQueueRequestsAsyncIterable<RequestQueueClientListRequestsResult> {
```